packages/backend/src/peripherals/database/shared/migrations.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module 'knex/lib/migrations/migrate/sources/fs-migrations' {
```

Comment:
should .d.ts file be tracked with git?

Reply:
Yes, because it is a custom .d.ts file hand written by me. The @types/knex pacakage doesn't include typings for this because it's only used as a workaround to support .js/.ts migrations